src/Aspire.Dashboard/Otlp/Model/OtlpInstrument.cs

```diff
@@ -27,6 +27,26 @@ public class OtlpInstrumentData
     public required OtlpInstrumentSummary Summary { get; init; }
     public required List<DimensionScope> Dimensions { get; init; }
     public required Dictionary<string, List<string?>> KnownAttributeValues { get; init; }
+
+    public static bool HasOverflowDimension(List<DimensionScope> dimensions)
+    {
+        // See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute
+        foreach (var dimension in dimensions)
+        {
+            // The spec says "otel.metric.overflow" should be the only attribute on the dimension.
+            // However, by this point there are other attributes in the data so we can't tell.
```

Comment:
is the other attributes coming from Instrument.Tags or Meter.Tags?

Reply:
No, there is a process to denormalize data in dimensions that the dashboard stores. It's kind of weird and I'm not sure if it's correct. And it's lacking in comments.

But the end result is there are more attributes mixed in here. Perhaps tags from meter/instrument would also add attributes.

Reply:
I changed the detection logic to happen when data is received rather than when it's displayed. Now it's accurate to the spec.